prisma/seed.ts

```diff
@@ -0,0 +1,46 @@
+import {PrismaClient} from '@prisma/client';
+import * as bcrypt from 'bcrypt';
+
+import {AUTH_CONSTANTS} from 'src/constants/auth-constants';
+
+const prisma = new PrismaClient();
+
+async function main() {
+  const password = await bcrypt.hash(
+    'Test!123',
+    AUTH_CONSTANTS.ROUNDS_OF_GASHING,
+  );
+
+  const user1 = await prisma.user.upsert({
+    where: {email: 'test1@test.test'},
+    update: {},
```

Comment:
I guess you don't need it here

Reply:
@skarm61 
It is required
https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#upsert
<img width="1000" alt="image" src="https://github.com/heorhii-ms/back-end-mentee/assets/82723713/3a0a4c43-93d2-49fe-a994-009f9896f497">
<img width="727" alt="image" src="https://github.com/heorhii-ms/back-end-mentee/assets/82723713/3e340011-8f17-4d04-9993-a53d15f3c8bc">

Reply:
but did you try without it? 
Could it be that only one of them in the same moment required

Reply:
@skarm61 
Yes, I tried.
No, both of them required.